packages/features/bookings/Booker/Booker.tsx

```diff
@@ -327,7 +327,7 @@ const BookerComponent = ({
                 {!hideEventTypeDetails && orgBannerUrl && !isPlatform && (
                   <img
                     loading="eager"
-                    className="-mb-9 h-28 max-h-28 rounded-tl-md sm:max-h-24"
```

Comment:
Does this fix the org banner responsive bug?

Reply:
https://github.com/calcom/cal.com/issues/14937

Reply:
Not sure if we should remove max height as user can upload banner with bigger height (even though we mention the recommended format is 1500x500)